buckets-mdapi/tests/rpc_handlers.rs

```diff
@@ -66,7 +66,7 @@ fn verify_rpc_handlers() {
     let pg_port = pg_url.port().expect("failed to parse postgres port");
     let pg_db = "test";
     let user = "postgres";
-    let application_name = "boray-test";
+    let application_name = "buckets_mdapi_test";
```

Comment:
hyphen changed to underscores here, no idea if it makes any difference, if not ignore this.

Reply:
Postgres seems generally more accepting of underscores so that's the only reason for the change.